frontend/src/views/ProblemPage.tsx

```diff
@@ -97,7 +60,7 @@ function ProblemPage() {
     setLoading(true);
     setError('');
 
-    editProblem(newProblem.problemId, newProblem)
+    editProblem(newProblem.problemId, newProblem, token || '')
```

Comment:
Yeah it would fail - I'd probably be ok with it since the server will return a proper error message anyways, but I went ahead and added an if statement anyways (that just says try again in a few seconds or refresh, since this would likely be an external failure rather than our own codebase's fault)

Reply:
I will note with `setError('An error occurred fetching your credentials; '        + 'please try again in a few seconds or refresh the page.');` that refreshing the page will likely cause the user to lose their updates? That may want to me noted in this error, so the user doesn't accidentally lose their work.

Reply:
That's a good catch - I'll just change the message to be 'please try again in a few seconds'